eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -11,7 +11,8 @@ param (
   $PublicArtifactLocation = "",
```

Comment:
Should we remove the $Language parameter? Or at least stop passing it in the template?

Reply:
I don't understand what you mean.

Reply:
This script takes a Language parameter but now Language is setup and coming from common.ps1 and in fact we might not even need to Language in this script any more since we are using the function we define in the language settings.

Reply:
Ok, I see what you mean, I'll do that.

Reply:
What is your plan for Language? Do you plan to just let common.ps1 override it for now and then remove it from the script once you can update the language repos to stop passing it?